src/main/java/Duke.java

```diff
@@ -1,10 +1,165 @@
-public class Duke {
```

Comment:
Perhaps consider outsourcing more task into other classes. The main class should not contain too many operations